src/null-safety/index.md

```diff
@@ -3,39 +3,39 @@ title: Sound null safety
 description: Information about Dart's null safety feature
 ---
 
-The Dart language comes with sound null safety. 
-
-In Dart 3, you always get null safety. In Dart 2.x,
-it must be enabled [with a pubspec setting](#enable-null-safety).
+The Dart language enforces sound null safety.
 
 Null safety prevents errors that result from unintentional access
 of variables set to `null`.
+
 For example, if a method expects an integer but receives `null`,
-your app causes a runtime error. This type of error, a null dereference error,
-can be difficult to debug.
-
-With sound null safety variables are 'non-nullable' by default:
-They can be assigned only values of the declared type
-(e.g. `int i=42`), and never be assigned `null`.
-You can specify that a type of a variable is nullable
-(e.g. `int? i`),
-and only then can they contain either a `null` *or*
+your app causes a runtime error.
+This type of error, a null dereference error, can be difficult to debug.
+
+With sound null safety, all variables require a value.
+This means Dart considers all variables _non-nullable_.
+You can assign values of the declared type only, like `int i=42`.
+You can never assign a value of `null` to default variable types.
+To specify that a variable type can have no value, add a `?` after
```

Comment:
We definitely shouldn't describe a variable as having "no value". `null` is most definitely a value. Consider:

```dart
var thing = null;
var stuff = [1, thing, 'three']);
print(stuff.length); // 3.
```

If `null` isn't a value, then what's the second element in this list?

Even before null safety, Dart ensured that all variables had a value. To ensure that, it would default initialize everything to `null`.

What null safety does is prevent you from *doing stuff* (accessing members) on `null` that it doesn't support.